### ISSUE TITLE
Reversal of Issues #15-22, #26

### ISSUE THREAD

**Issue by tacticalnoot** [stellar-expert/public-directory#636]
[Issue 17](https://github.com/stellar-expert/public-directory/issues/17)

Issues 15-22, 26 appear to be false reporting/libel from one account. 

Requesting these domains be reinstated.

**Comment by CryptoHobbyist** [comment]
@tacticalnoot 

I would agree that it appears that the @djolehaubica reports seem to be from a supporter (or DEV) of the QSI scam minters and reported a few tokens (useful or not) that were know to be minted by people that were very active in exposing the scammers behind the QSI scam and laundering system.

It is up to @orbitlens to review these domains flagged as being a scam and determine what changes, if any, need to be made. Scam, speculative, ill thought out, dead, counterfeit, private pooling system, or abandoned are definately different.

Rob TheCrypto Hobbyist

**Comment by tacticalnoot** [comment]
[Blocklist.txt](https://github.com/stellar-expert/public-directory/blob/cc2670b8e00c0c5adf40c5084de26bf39ce2e9a5/domains/blocklist.txt) appears to be the location of the affected domains for the requested edits; the result of a “revenge request”.  
Lines 93-99, 112

**Comment by tacticalnoot** [comment]
Hello, 

It has been nearly 1 month since this request was made and there have been no actions taken. 

TREAD token was able to move their domain to another website and be listed because their supply and token are unlocked. 

There are other projects in this request that are locked to StellarMint and cannot be moved. 

Please clear these domains so that we can be listed.

Thank you.

**Comment by tacticalnoot** [comment]
Hi OrbitLens!

I’m commenting to follow up this issue and request that the requested domains be removed from your blocklist. 

Since making this request I have made additional efforts to familiarize and socialize with the Stellar Development Community and Stellar Global. 

I have been a community member of Stellar for over 7 years now, and intend to stay with this community.

In addition to this, I post liquidity reports and proof-of-reserves for the projects I’m working on multiple times per month. 

Thank you for your consideration.

**Comment by tacticalnoot** [comment]
Dear OrbitLens, 

I am writing again to follow up on this request, with a more specific inquiry to clear the tokens listed on https://gqxamm.com 

The other tokens and domains listed outside of host KingTroll.stellarmint.iO are not of my concern despite djolehaubicas blanket attack on them. 

All I ask is that GQX and AMM be allowed to have aggregator presence on your platform. 

Please and thank you, 
Noot
